lib/components/user/saved-trip-screen.js

```diff
@@ -215,6 +239,7 @@ const mapStateToProps = (state, ownProps) => {
   const activeItinerary = activeSearch && activeSearch.activeItinerary
   const itineraries = getActiveItineraries(state.otp) || []
   return {
+    activeSearchId: state.otp.activeSearchId,
```

Comment:
Unnecessary if we move logic to actions.

Reply:
Updated in https://github.com/opentripplanner/otp-react-redux/pull/232/commits/861f5e8d951548dac61c07d5083f2b60682f604a. Previous logic is no longer necessary.